src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+  const sevenMoreOff = 50;
+  const threeMoreOff = 20;
+
+  if (days >= 7) { return totalCost - sevenMoreOff; }
+  if (days >= 3) { return totalCost - threeMoreOff; }
+  return totalCost;
```

Comment:
need new line before `return`
```suggestion

  return totalCost;
```